addons/l10n_id_efaktur/models/account_move.py

```diff
@@ -38,19 +37,35 @@ class AccountMove(models.Model):
         ], string='Kode Transaksi', help='Dua digit pertama nomor pajak',
         readonly=False, copy=False,
         compute="_compute_kode_transaksi", store=True)
+    l10n_id_efaktur_range = fields.Many2one("l10n_id_efaktur.efaktur.range", string="E-faktur Range", copy=False, domain="[('company_id', '=', company_id), ('available', '>', 0)]")
     l10n_id_need_kode_transaksi = fields.Boolean(compute='_compute_need_kode_transaksi')
+    l10n_id_available_range_count = fields.Integer(compute="_compute_available_range_count")
+
+    def _compute_available_range_count(self):
```

Comment:
```suggestion
    @api.depends('company_id')
    def _compute_available_range_count(self):
```
Otherwise, the compute doesn't get trigger until you save the record, and the field stays hidden